docs/music.rst

```diff
@@ -95,18 +94,14 @@ Functions
     their defaults before the music (whatever it may be) is played.
 
     An optional argument to specify the output pin can be used to override the
-    default of ``microbit.pin0``. With micro:bit **V2** you can use
-    ``microbit.pin_speaker`` or a tuple of two pins, for example
-    ``pin=(pin_speaker, pin0)`` which would output sound on the built-in
-    speaker and pin 0.
+    default of ``microbit.pin0``.
 
     If ``wait`` is set to ``True``, this function is blocking.
 
     If ``loop`` is set to ``True``, the tune repeats until ``stop`` is called
     (see below) or the blocking call is interrupted.
 
-.. py:function:: pitch(frequency, duration=-1, pin=microbit.pin0, wait=True)
-                 pitch(frequency, duration=-1, pin=(pin_speaker, pin0), wait=True)
+.. py:function:: pitch(frequency, duration=-1, pin=pin0, wait=True)
```

Comment:
pin argument is not mentioned in this description.

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/695/commits/05c8b169faa12a8b27fa760a3f374dd47bc999a1